src/App.tsx

```diff
@@ -15,48 +17,108 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+export enum SortType {
+  Alphabet = 'abc',
+  Length = 'length',
+  Default = '',
+}
+
+function getSortedGoods(
+  goods: string[],
+  sortField: SortType,
+  isReversed: boolean,
+): string[] {
+  const sortedGoodsCopy = [...goods];
+
+  if (sortField === SortType.Alphabet) {
+    sortedGoodsCopy.sort((good1, good2) => {
+      return good1.localeCompare(good2);
+    });
+  }
+
+  if (sortField === SortType.Length) {
+    sortedGoodsCopy.sort((good1, good2) => {
+      return good1.length - good2.length;
+    });
+  }
```

Comment:
Its better to use swich-case

Reply:
I tried to do it with `switch-case`, but I prefer to write it with `if`, because in this case we have only one return of the default meaning instead of two (the default after switch-case and the other one after reversed condition). That is why I decided to stay with if-statement.